Resources/Prototypes/_White/Catalog/uplink_catalog.yml

```
@@ -108,3 +108,14 @@
   saleLimit: 1
+
+- type: listing
+  id: UplinkCrateNPCGondola
+  name: uplink-gondola-name
+  description: uplink-gondola-desc
+  icon: { sprite: /Textures/_White/Mobs/Animals/gondola.rsi, state: icon }
+  productEntity: CrateNPCGondola
+  cost:
+    Telecrystal: 2
+  categories:
+  - UplinkUtility
```

yamllint

[warning] wrong indentation: expected 4 but found 2

(indentation)